src/buildHTML.js

```diff
@@ -770,18 +748,18 @@ groupTypes.spacing = function(group, options) {
 
 groupTypes.llap = function(group, options) {
     const inner = makeSpan(
-        ["inner"], [buildGroup(group.value.body, options.reset())]);
+        ["inner"], [buildGroup(group.value.body, options)]);
```

Comment:
No: The `.reset()` isn't necessary any more because `Options` objects are immutable.